.github/workflows/new-version.yml

```diff
@@ -0,0 +1,67 @@
+name: Create Tag for JS Package Version Bump
+
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - 'package.json'
+  workflow_dispatch:
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
+      tag-exists: ${{ steps.check-tag.outputs.exists }}
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Extract version from package.json
+        id: get-version
+        run: |
+          VERSION=$(jq -r .version package.json)
+          echo "Extracted version is $VERSION"
+          echo "::set-output name=version::$VERSION"
+          
+      - name: Check if tag exists
+        id: check-tag
+        run: |
+          if git tag | grep -q "v${{ steps.get-version.outputs.version }}"; then
+            echo "Tag exists"
+            echo "::set-output name=exists::1"
```

Comment:
The `::set-output` command is deprecated in GitHub Actions. Use this method instead: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

Reply:
Context: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/